namematch/block.py

```diff
@@ -1460,21 +1460,24 @@ def load_main_index_nn_strings(og_blocking_index_file):
     return main_index_nn_strings
 
 
-def write_some_cps(cand_pairs, candidate_pairs_file, header=False):
+def write_some_cps(cand_pairs, candidate_pairs_file):
     '''Write out a portion of the candidate-pairs.
 
     Args:
         cand_pairs (pd.DataFrame): chunk of the candidate-pairs file
         candidate_pairs_file (str): path to the candidate-pairs file
         header (bool): True if this is the first time calling this function
```

Comment:
Can you please remove the `header` parameter from the docstring and then rebuild the docs?